file/hasher/param.go

```diff
@@ -0,0 +1,56 @@
+package hasher
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ethersphere/swarm/file"
+)
+
+// defines the boundaries of the hashing job and also contains the hash factory functino of the job
+// setting Debug means omitting any automatic behavior (for now it means job processing won't auto-start)
+type treeParams struct {
+	SectionSize int
```

Comment:
can we maybe once and for all define these terms? i'm not sure how i'm supposed to review this
please explain section size, branches and spans

Reply:
Updated the PR description.

Reply:
thanks hun